CONTRIBUTING.md

```diff
@@ -5,12 +5,10 @@ project really) strong and successful.
 
 # Legal
 
-All original contributions to Hibernate are licensed under the 
-[GNU Lesser General Public License (LGPL)](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt), 
-version 2.1 or later, or, if another license is specified as governing the file or directory being 
-modified, such other license.
-
-The LGPL text is included verbatim in the [lgpl.txt](lgpl.txt) file in the root directory of the ORM repository.
+All original contributions to Hibernate are licensed under the
+[Apache License version 2.0 (Apache-2.0)](https://www.apache.org/licenses/LICENSE-2.0.txt),
+or, if another license is specified as governing the file or directory being modified, such other license.
+The Apache-2.0 license text is included verbatim in the [`LICENSE.txt`](LICENSE.txt) file
```

Comment:
```suggestion
All original contributions to Hibernate are licensed under the
[Apache License version 2.0 (Apache-2.0)](https://www.apache.org/licenses/LICENSE-2.0.txt),
or, if another license is specified as governing the file or directory being modified, such other license.
The Apache-2.0 license text is included verbatim in the [`LICENSE.txt`](LICENSE.txt) file

Note that Hibernate ORM 7.0.0.Beta4 and earlier are distributed under [a different license](https://github.com/hibernate/hibernate-orm/blob/6.6/CONTRIBUTING.md#legal). To allow for backports, the Hibernate team may ask contributors to dual-license their contribution.
```

Reply:
Hm, I think I'd change "the Hibernate team may ask ..." to "the Hibernate team asks ..." and leave the PR banner about dual-licensing.

Reply:
Sure, makes sense